packages/common-types/rollup.config.js

```diff
@@ -0,0 +1,18 @@
+import typescript from "rollup-plugin-typescript2";
```

Comment:
Does this really need to be rolled up?

Only functionality we want from this package is effectively the `.d.ts` files, which are built with the tsc compiler, no?

Reply:
Yup, it breaks without it. Believe me i didn't want to.

Reply:
Specifically the jest module resolver breaks.

Also we need it regardless because common-types has to expose enum values

Reply:
Hmm alright